source/backend/api/Services/CompensationRequisitionService.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
+using DocumentFormat.OpenXml.Office2010.Excel;
```

Comment:
This needs to be removed (seems like you are sharing the same branch from your last pr?)

Reply:
removed